mlrun/runtimes/local.py

```diff
@@ -486,4 +486,17 @@ def get_func_arg(handler, runobj: RunObject, context: MLClientCtx, is_nuclio=Fal
                 kwargs[key] = obj.local()
             else:
                 kwargs[key] = context.get_input(key, inputs[key])
+
+    # VAR_KEYWORD meaning : A dict of keyword arguments that aren’t bound to any other parameter.
+    # This corresponds to a **kwargs parameter in a Python function definition.
+    if any(param.kind == param.VAR_KEYWORD for param in args.values()):
+        # if handler has **kwargs, pass all parameters provided by the user to the handler which were not already set
+        # as part of the previous loop which handled all parameters which were explicitly defined in the handler
+        for key in params:
+            if key not in kwargs:
+                kwargs[key] = copy(params[key])
+        for key in inputs:
+            if key not in kwargs:
+                kwargs[key] = context.get_input(key, inputs[key])
```

Comment:
The input processing above has more to it (using `obj.local()` in some cases) - maybe have that piece of logic in an internal function, and use it here as well. Or did you verify this is dead code in which case I would remove it there as well?